build.gradle.kts

```diff
@@ -36,24 +24,26 @@ publishing {
             }
         }
     }
+    publications.create("default", MavenPublication::class.java) {
+        from(components.getByName("java"))
+        artifact(tasks.register("emptySources", Jar::class.java) {
+            archiveClassifier = "sources"
+        })
+        artifact(tasks.register("emptyDocs", Jar::class.java) {
+            archiveClassifier = "javadoc"
+        })
+
+        groupId = project.rootProject.group.toString()
+        version = project.rootProject.version.toString()
+        artifactId = project.name
```

Comment:
Just wondering: are those options really needed?
AFAIK, `groupId`, `artifactId`, `version` derive from the current project.
Why configuring the options here explicitly?

Would it be better to configure the project's groupId, version, name accordingly instead?

Reply:
This code wasn't used, I removed it [here](https://github.com/GradleUp/nmcp/pull/29/commits/4e7a93d7972f5af96cb1ac92c7e1d915db9f65e5)